frontend/src/app/app.component.ts

```diff
@@ -0,0 +1,159 @@
+  consoleOutput: LogOutput[] = [];
+
+  constructor(private http: HttpClient) {}
+
+  /**
+    * Sends a request to either the database service or the price service corresponding to the selected selectedDestination, selected in the ui.
+
+  */
+  async sendRequest() {
+    if (this.selectedDestination == this.dbDestination) {
+      if (this.requestName === '' || this.requestName === 'account-worth') {
+        this.getRequestDatabaseService(`${this.dbDestination}`).subscribe(
+          (data) => {
+            this.fetchedResult = data.result;
+            console.log(this.fetchedResult);
+            this.addResultToOutputLog(this.fetchedResult);
+          },
+          (error) => {
+            this.addErrorToOutputLog(error);
+          }
+        );
+      } else {
+        this.getRequestDatabaseService(
+          `${this.dbDestination}request-handler/`
+        ).subscribe(
+          (data) => {
+            this.fetchedResult = data.result;
+            this.addResultToOutputLog(this.fetchedResult);
+          },
+          (error) => {
+            this.addErrorToOutputLog(error);
+          }
+        );
+      }
+    } else {
+      this.getRequestPricesService(
+        `${this.priceDestination}request/`
+      ).subscribe(
+        (data) => {
+          console.log(data);
+          this.fetchedResult = data.result.result;
+          this.addResultToOutputLog(this.fetchedResult);
+        },
+        (error) => {
+          this.addErrorToOutputLog(error);
+        }
+      );
+    }
+  }
+
+  /**
+   * Creates a "log" that will be displayed in the ui
+   *
+   * @param message that should be displayed
+   */
+  printRequestType(message) {
+    this.consoleOutput.push({
+      message: message,
+      type: 'info',
+    });
+  }
+
+  /**
+   * Sends a get request to the database service
+   *
+   * @param url of the get request
+   */
+  getRequestDatabaseService(requestServiceUrl: string): any {
+    console.log('hi');
+    return this.http.post(this.backendDestination, {
+      requestService: requestServiceUrl,
+      endpoint: this.requestName,
+    });
+  }
+
+  addResultToOutputLog(result) {
+    this.consoleOutput.push({
+      message: `Successful, Result: ${result}`,
+      type: 'success',
+    });
+  }
+
+  addErrorToOutputLog(error) {
+    this.consoleOutput.push({
+      message: 'Request failed',
+      type: 'error',
+    });
+    this.handleError('Price Service', error);
+  }
+
+  /**
+   * Sends a get request to the price service
+   *
+   * @param url of the get request
+   */
+  getRequestPricesService(requestServiceUrl: string): any {
+    return this.http.post(this.backendDestination, {
+      requestService: requestServiceUrl,
+      endpoint: this.requestName,
+    });
+  }
+
+  /**
+    reports an error to the error monitor
+    
+    @param source the source of the error
+    @param error the error that occurred, if the error already has a correlationId, it will be used when reporting this error
+  */
```

Comment:
einheitlicher still der kommentare ist schöner.
```suggestion
  /**
   * reports an error to the error monitor
   * 
   * @param source the source of the error
   * @param error the error that occurred, if the error already has a correlationId, it will be used when reporting this error
  */
```